sonar-core/src/main/java/org/sonar/core/technicaldebt/DefaultTechnicalDebtModel.java

```diff
@@ -38,6 +39,7 @@
 
 import static com.google.common.collect.Lists.newArrayList;
 
+@RequiresDB
 public class DefaultTechnicalDebtModel implements TechnicalDebtModel {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)"TechnicalDebtModel" is deprecated, implement the suggested replacement instead.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1724)